### PR TITLE
Updated the Tutorial

### PR DESCRIPTION
**Pull request by Om-Pandey**
The current version has the following updates:

- Dataset integration verified
- Attention mechanism added
- tfa.seq2seq is used

Solves queries added in #986

### CODE REVIEW

**Comment by Om-Pandey** [comment]
@seanpmorgan please review and add this, the changes proposed in #986 have been made

File: docs/tutorials/networks_seq2seq_nmt.ipynb [hunk 1]
```diff
@@ -5,34 +5,21 @@
     "colab": {
```
Comment (seanpmorgan) [inline docs/tutorials/networks_seq2seq_nmt.ipynb]: <p>nit: 5 essential topics?</p><p>nit: typo architecture</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/1003/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (seanpmorgan) [inline docs/tutorials/networks_seq2seq_nmt.ipynb]: <p>Mind keeping the output of this cell. I think it's a nice way for users to scroll to the bottom and see a working translation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/1003/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

**Comment by Om-Pandey** [comment]
@seanpmorgan did the changes, yeah ... I mentioned the problems with TF 2.1 installation earlier too.
Anyways, if there is anything required LMK! Thanks!

**Pull request merged**